docs/dashboards/overview-dashboard.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[overview-dashboard]]
+= Overview dashboard
+
+The Overview dashboard provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation.
+
+image::images/overview-pg.png[Overview dashboard]
+
+[discrete]
+== Live feed
+
+From the live feed on the Overview dashboard, you can quickly access recently created cases, favorited timelines, and the latest {elastic-sec} news.
+
+TIP: The *Security news* section provides the latest {elastic-sec} news to stay informed on new developments, learn about {elastic-sec} features, and more.
+
+image::images/live-feed-ov-page.png[Overview dashboard with live feed section highlighted]
+
+[discrete]
+== Histograms
+
+Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on areas of interest in time-based histograms, select a region to reflect a date range, or select a preset value in the timepicker. In the *Stack by* dropdown, you can select specific parameters to visualize individual counts. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display the total counts by alert name within the specified time frame.
+
+TIP: All Elastic Security histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.
+
+[discrete]
+== Host and network events
+
+View event and host counts specific to Elastic data shippers and apps, such as *Auditbeats* or *Elastic Endpoint Security*. Expand each category to view specific counts of hosts or network events related to the selected category.
```

Comment:
```suggestion
View event and host counts specific to Elastic data shippers and apps, such as *Auditbeats* or *{endpoint-cloud-sec}*. Expand each category to view specific counts of hosts or network events related to the selected category.
```

Reply:
Not sure if this is correct, so ignore if it's not!

Reply:
It's correct, and probably updated in another PR for the renaming of Endpoint Security, but might as well make sure it's updated here too.